lib/portlayer/exec/handle.go

```diff
@@ -244,9 +251,8 @@ func Create(ctx context.Context, sess *session.Session, config *ContainerCreateC
 	}
 
 	specconfig := &spec.VirtualMachineConfigSpecConfig{
-		// FIXME: hardcoded values
-		NumCPUs:  2,
-		MemoryMB: 2048,
+		NumCPUs:  int32(config.Resources.NumCPUs),
```

Comment:
It comes in as int64 from docker. We have to cast it somewhere, I figured that was the right spot.